src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java

```diff
@@ -356,7 +358,7 @@ public void release() {
     // }
   }
 
-  volatile Object lock = new Object();
+  transient volatile Object lock = new Object();

```

Comment:
Should probably use an actual `Lock` object instead of synchronizing on a regular object. Synchronization is kind of an older pattern no longer recommended